manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Minion.MMOUI

```

```suggestion
PackageIdentifier: GoodGameModsLLC.Minion
```

Please update file name and path to match this change.